tests/kafkatest/services/performance/performance.py

```diff
@@ -19,22 +19,24 @@
 
 class PerformanceService(KafkaPathResolverMixin, BackgroundThreadService):
 
-    def __init__(self, context, num_nodes, stop_timeout_sec=30):
+    def __init__(self, context, num_nodes, process_pattern="", root="/mnt/*", stop_timeout_sec=30):
```

Comment:
Hmm... Shouldn't we be overriding `process_pattern` in extensions?

Reply:
If `process_pattern` is the empty string, we get the old behavior where all java processes are killed.  The idea behind making this the default was to avoid breaking subclasses that haven't been updated (because they don't live in this git repo).  Later, we can remove the default once everything has been updated.

Reply:
Makes sense. Maybe we can add a comment to that effect since it seems not obvious?